clients/unshaded/src/main/java/tachyon/client/file/options/InStreamOptions.java

```diff
@@ -50,6 +55,16 @@ public Builder(TachyonConf conf) {
       ReadType defaultReadType =
           conf.getEnum(Constants.USER_FILE_READ_TYPE_DEFAULT, ReadType.class);
       mTachyonStorageType = defaultReadType.getTachyonStorageType();
+      try {
+        mLocationPolicy =
+            CommonUtils
+                .createNewClassInstance(
+                    ClientContext.getConf().<FileWriteLocationPolicy>getClass(
+                        Constants.USER_FILE_WRITE_LOCATION_POLICY),
+                    new Class[] {}, new Object[] {});
+      } catch (Exception e) {
```

Comment:
is it because you want to wrap any exception the try block can call as RuntimeException? ... meaning you crash the client when there is invalid write location policy?